OrbitGl/TextBox.h

```diff
@@ -67,6 +67,9 @@ class TextBox
 
     void SetTextY( float a_Y ){ m_TextY = a_Y; }
 
+    void SetElapsedTimeTextLength( const size_t& a_Length ){ m_ElapsedTimeTextLength = a_Length; }
```

Comment:
a_Length should be passed by value as it will be passed through register.

Reply:
grrr, ya, fudged this also :(